frontend/src/styles/global.css

```
@@ -4,6 +4,19 @@
  * Inspired by Linear, Anthropic, Stripe - editorial quality with functional clarity
  */
 
+@font-face {
+  font-family: "Fraunces";
```

Stylelint (17.14.0)

[error] Expected no quotes around "Fraunces" (font-family-name-quotes)

(font-family-name-quotes)